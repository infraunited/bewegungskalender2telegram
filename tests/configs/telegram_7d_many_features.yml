output:
  type: 'telegram'
  stdout: true
caldav:
  url: 'http://127.0.0.1:18476'
  calendars:
    - url: '/d/5cbae8f0-5881-c7a2-4cbb-c5c1ba05bc25/'
      name: 'Test calendar 1'
    - url: '/d/701e875a-44ca-97df-0140-bbb36dc26a9b/'
  username: 'x'
  password: 'x'
  offset_days: 2
  number_days: 7
  current_day_override: "2022-12-29"
format:
  time_locale: "de_DE.utf8"
  timezone: "Europe/Berlin"
  show_calendar_name: true
  show_location: true
  show_description: true
  cutoff_description: 0
  header_txt: "*Was passiert die nächsten Tage?*"
  footer_txt: >
    _Diese Nachricht wurde automatisch generiert\.
    Im [Kalender](http://caldav.example)
    können Termine hinzugefügt und geändert werden\._
