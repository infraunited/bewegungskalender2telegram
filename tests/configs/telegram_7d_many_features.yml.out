*Was passiert die nächsten Tage?*


📅 *__Samstag__ 2022\-12\-31*

*Multi\-All\-Day* \(Test calendar 1\)
🌍 _with location_
and description


📅 *__Sonntag__ 2023\-01\-01*

*Multi\-All\-Day* \(Test calendar 1\)
🌍 _with location_
and description

*Single All\-Day* \(Test calendar 1\)

__08:30__ *Long timed event* \(testing2\)
🌍 _somewhere_
here is the description
of this
beautiful
event

__09:00__ *Single Event* \(testing2\)


📅 *__Montag__ 2023\-01\-02*

*Multi\-All\-Day* \(Test calendar 1\)
🌍 _with location_
and description

*Long timed event* \(testing2\)
🌍 _somewhere_
here is the description
of this
beautiful
event

__10:00__ *\-\>test \[s@\_\+\}* \(Test calendar 1\)
🌍 _\]\|🔋/\~ \*&^'_
"\#no\`for\.\!"\)\=\(


📅 *__Dienstag__ 2023\-01\-03*

*?* \(Test calendar 1\)

*Multi\-All\-Day* \(Test calendar 1\)
🌍 _with location_
and description

__00:00__ *Long timed event* \(testing2\)
🌍 _somewhere_
here is the description
of this
beautiful
event

__00:00__ *Recurring twice weekly* \(testing2\)
🌍 _anywhere\.\.\._
a weekly event should not have too much description, people will get tired of reading it every time

__00:00__ *?* \(testing2\)

__07:30__ *An event without time* \(testing2\)


📅 *__Mittwoch__ 2023\-01\-04*

*Multi\-All\-Day* \(Test calendar 1\)
🌍 _with location_
and description


📅 *__Donnerstag__ 2023\-01\-05*

*Midnight to following day* \(testing2\)


📅 *__Freitag__ 2023\-01\-06*

__00:00__ *Midnight to following day* \(testing2\)

__00:00__ *Recurring twice weekly* \(testing2\)
🌍 _anywhere\.\.\._
a weekly event should not have too much description, people will get tired of reading it every time


_Diese Nachricht wurde automatisch generiert\. Im [Kalender](http://caldav.example) können Termine hinzugefügt und geändert werden\._